lxd/api_metrics.go

```diff
@@ -4,14 +4,14 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/canonical/lxd/lxd/auth"
```

Comment:
That's weird, I have the linter running on a pre-push hook, I'm not sure why it didn't catch this.